CHANGELOG.md

```diff
@@ -32,6 +32,15 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 -
 
 
+## [0.1.59] - 2022-11-16
+
+### Internal Changes
+
+- OpenShift platforms were added as an annotation. This helps organize
+  operators across the OpenShift ecosystem.
+- The `preamble.json` file was included in release targets, making it easier to
+  include changes when releasing new versions of the operator.
+
 ## [0.1.58] - 2022-11-14
```

Comment:
Not sure if this needs to be changed to 0.1.59. Otherwise, looks good!

Reply:
Nope, 0.1.58 already went out the door, so we need to cut a new release.